src/store/modules/sitestatus/index.js

```diff
@@ -267,6 +273,14 @@ const mutations = {
     state.obs_settings = status.obs_settings
   },
 
+  new_owmReport (state, newOwmReport) {
```

Comment:
the function name here could be more descriptive - I think `storeNewOwmReport` would make more sense